packages/react/src/__tests__/jest-matcher.test.tsx

```diff
@@ -1,6 +1,7 @@
-import { styled } from '@compiled/react';
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx, styled } from '@compiled/react';
```

Comment:
Is there a way of enabling local namespaces without importing `jsx` and just using the pragma?

Reply:
Using `jsxImportSource` and the automatic runtime 👍

Reply:
Yeah, so can we do that for this test?

Reply:
Ah nice, sorted